docs/01_setup.md

```diff
@@ -71,18 +71,30 @@ You can read about the [MACI requirements here](https://maci.pse.dev/docs/instal
 ```bash
 git clone https://github.com/privacy-scaling-explorations/maci.git && \
 cd maci && \
+git checkout v1.2.2 && \
 pnpm i && \
 pnpm run build
 ```
 
+> [!IMPORTANT]
+> It's important to use version 1.2.2 of MACI, as this version's circuit are audited and have zKeys which have undergone a trusted setup.
+
 ### Download .zkey files
 
-Note the locations of the .zkey files as the CLI requires them as command-line flags. Download ceremony artifacts:
+Download ceremony artifacts for production:
 
 ```bash
 pnpm download:ceremony-zkeys
 ```
 
+or the test keys for testnet only:
+
+```bash
+pnpm download:test-zkeys
```

Comment:
do we need to specify this?
```
pnpm download:test-zkeys-1-3
```

Reply:
https://github.com/privacy-scaling-explorations/maci-rpgf/pull/110#discussion_r1609485727

Reply:
also, gonna open a pr soon to remove 1-3/1-2 from the download command and have only one

Reply:
sorry missed that, thanks!